addons/mail/static/src/model/misc.js

```diff
@@ -67,3 +68,12 @@ export function isRelation(Model, fieldName) {
 export function isFieldDefinition(val) {
     return val?.[FIELD_DEFINITION_SYM];
 }
+
+/**
+ * @template T
+ * @param {T} obj
+ * @returns {T}
+ */
+export function getPlainReactive(obj) {
+    return toRaw(obj)?._proxy ?? reactive(obj);
+}
```

Comment:
I still believe this method should just be `toRaw`, and if there are bugs we should investigate and fix them (internal code should always do writes through internal reactive).